linera-views/src/context.rs

```diff
@@ -306,7 +306,25 @@ where
     }
 }
 
-impl<E> ContextFromStore<E, MemoryStore> {
+/// An implementation of [`crate::context::Context`] that stores all values in memory.
+pub type MemoryContext<E> = ViewContext<E, MemoryStore>;
+
+/// Provides a `MemoryContext<()>` that can be used for tests.
+/// It is not named create_memory_test_context because it is massively
+/// used and so we want to have a short name.
```

Comment:
That's probably obsolete?

(`create_test_memory_context` is no shorter than `create_memory_test_context`. :smile:)

Reply:
Right, I meant to remove this